aeon.ttl

```diff
@@ -1,8 +1,14 @@
 @prefix : <https://github.com/tibonto/aeon#> .
 @prefix dc: <http://dublincore.org/specifications/dublin-core/dcmi-terms/2012-06-14/> .
 @prefix ns: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
-@prefix bfo: <http://purl.obolibrary.org/obo/bfo/2020/bfo.owl#> .
-@prefix iao: <http://purl.obolibrary.org/obo/iao/2020-06-10/iao.owl#> .
```

Comment:
This is where IAO was pinned to a specif version @StroemPhi

Reply:
Hence bfo prefix will become
`@prefix bfo: <https://standards.iso.org/iso-iec/21838/-2/ed-1/en/owl/bfo-2020.owl> .`